app/classes/amity.py

```diff
@@ -1,5 +1,6 @@
 import random
```

Comment:
Maybe have a Readme file, requirements.txt files. 

Also the naming of your branches, try to name clearly next time.